src/main/java/by/exadel/internship/repository/UserRepository.java

```diff
@@ -40,4 +41,7 @@
     List<User> findAllWithSkillByInternshipId(@Param("id") UUID internshipId, @Param("role") UserRole role);
 
     List<User> findAllByUserRole(UserRole userRole);
+
+    @Query(value = "SELECT * FROM user_detail as u JOIN user_skill as s ON u.u_id = s.us_u_id WHERE CAST (s.us_name AS VARCHAR) IN :skills AND u.u_deleted = false", nativeQuery = true)
```

Comment:
Why native query?

Reply:
Because user_detail table don't have column skills and we neet to join and find by skill name, and skill is Enum, that is why i decided to create native querry.

Reply:
ok